src/components/SlideOver/SlideOver.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import './SlideOver.css'
+import { Button, Icon, Space, Transition, Typography } from '../../index'
+
+interface Props {
+  className?: string
+  children?: React.ReactNode
+  title?: string
+  description?: string
+  visible: boolean
+  wide?: boolean
+  loading?: boolean
+  align?: 'right' | 'left'
+  alignFooter?: 'right' | 'left'
+  hideFooter?: boolean
+  customFooter?: React.ReactNode
+  onCancel?: any
+  onCancelText?: string
+  onConfirm?: any
+  onConfirmText?: string
+}
+
+const SlideOver = ({
+  className = '',
+  children,
+  title,
+  description,
+  visible = true,
+  wide = false,
+  loading,
+  align = 'right',
+  alignFooter = 'right',
+  hideFooter = false,
+  customFooter = undefined,
+  onConfirm,
+  onCancel,
+  onConfirmText = 'Confirm',
+  onCancelText = 'Cancel',
+}: Props) => {
+  function stopPropagation(e: React.MouseEvent) {
+    e.stopPropagation()
+  }
+
+  const left = align === 'left'
+  const orientationClasses = left ? 'left-0 pr-10' : 'right-0 pl-10'
```

Comment:
@MildTomato are we aiming to extract all styles to individual classes or this can stay?

Reply:
i've tried to only use tailwind classes in css files - currently the library doesn't purge css for tailwind based on any of the .tsx/.js files so it's currently safer to just @apply tailwind classes in `SlideOver.css` and just use a css class name like `.sbui-slideover--left` in the actual .tsx file.

we need to figure out a better way to handle css, there's a few problems with how i've done it, it could be so much better :D - we could probably use css modules to prepend `sbui` onto classnames as well.